sourcegen-model/src/main/java/io/micronaut/sourcegen/model/ClassTypeDef.java

```diff
@@ -460,7 +460,7 @@ static ClassTypeDef of(ClassElement classElement) {
                 new ClassElementType(classElement, classElement.isNullable()),
                 classElement.getTypeArguments().entrySet()
                     .stream()
-                    .<TypeDef>map(e -> TypeDef.variable(e.getKey(), TypeDef.erasure(e.getValue())))
+                    .map(e -> TypeDef.erasure(e.getValue()))
```

Comment:
@dstepanov Not sure why it was a type variable before, but no of the tests seem to be failing by this fix